pandoc/references.bib

```diff
@@ -70,7 +70,7 @@ @misc{johnmyleswhiteJuliaDataDataFramesJl2020
 @misc{jump2021using,
   title = {JuMP Style Guide},
   year = 2021,
-  howpublished = {\url{https://jump.dev/JuMP.jl/stable/developers/style/#using-vs.-import}},
```

Comment:
Hardcoding versions?

Reply:
I just did that in 7acc2f9

Reply:
For the docs it makes sense, right?

Reply:
What if in year from now JUMP is on v1? Do we need to keep track of JuMP versions?

Reply:
It's only a ref to the information stored in that style guide. That information is still valid when JuMP is at a newer version.

Reply:
OK